packages/monitor-v2/src/health-check-runner/index.ts

```diff
@@ -26,10 +26,12 @@ export async function run(): Promise<void> {
 
   logger[outputLogLevel]({
     at: "HealthCheckRunner",
-    message: `Health check command ${outputLogLevel == "error" ? "failed" : "succeeded"}!`,
+    message: `Health check command ${outputLogLevel == "error" ? "failed 🩺" : "succeeded"}!`,
     validOutputs: validOutputs.map((output) => output.cmd),
-    errorOutputs,
+    errorOutputs: errorOutputs.map((outputs) => outputs.stderr),
```

Comment:
makes sense to limit this to stderr as stdout could be quite verbose in DVMv2 health check scripts